index.js

```diff
@@ -154,17 +160,25 @@ module.exports = {
       let ecOrigin = ecProtocol + ecHost + ':' + options.port;
 
       appendSourceList(policyObject, 'connect-src', ecOrigin);
-      appendSourceList(policyObjectForTest, 'connect-src', ecOrigin);
-
       policyObject['report-uri'] = ecOrigin + REPORT_PATH;
-      policyObjectForTest['report-uri'] = policyObject['report-uri'];
+
+      if (policyObjectForTest) {
+        appendSourceList(policyObjectForTest, 'connect-src', ecOrigin);
+        policyObjectForTest['report-uri'] = policyObject['report-uri'];
+      }
     }
 
     this._policyString = buildPolicyString(policyObject);
-    this._policyStringForTest = buildPolicyString(policyObjectForTest);
+
+    if (policyObjectForTest) {
+      this._policyStringForTest = buildPolicyString(policyObjectForTest);
+    }
 
     app.use((req, res, next) => {
-      let isRequestForTests = req.originalUrl.startsWith('/tests');
+      // Use policy for test environment if
+      // - the request is for tests and
+      // - the build include tests
```

Comment:
Minor, but could clarify:

```
      // Use policy for test environment if both of these conditions are met:
      // 1. the request is for tests and
      // 2. the build include tests
```

Reply:
Thanks a lot for your suggestion. Applied it in 952f3fccb4b9bf4b2d8a80cfd6b9991a25435eec.